ui.xml

```diff
@@ -229,10 +229,10 @@
 </Panel>
 <Panel id="panelUIToggle" active="false" width="22" height="22" rectAlignment="UpperRight" offsetXY="-402 0">
     <Panel id="panelUIToggleShow" visibility="">
-        <Button onClick="showButtons" fontSize="12" textAlignment="MiddleCenter">▼</Button>
+        <Button onClick="showButtons" fontSize="12" textAlignment="MiddleCenter">&#9660;</Button>
```

Comment:
with the latest changes in tabletop-tools i dont think this change is necessary anymore, also makes it a little harder since you have to know what the code is for

Reply:
Fair enough, though we could use an XML comment to indicate what it is (e.g. `<!-- up arrow -->`) Or I can revert it.  Up to you.